auth-web/src/components/auth/mixins/NextPageMixin.vue

```diff
@@ -81,25 +81,27 @@ export default class NextPageMixin extends Vue {
         }
         return `/${nextStep}`
       case LoginSource.BCEID:
-        let bceditNextStep = '/'
+        let bceidNextStep = '/'
         // Redirect to TOS if no terms accepted
         // for invited users , handle user profile
         // Redirect to create team if no orgs
         // Redirect to dashboard otherwise
         if (!this.userProfile?.userTerms?.isTermsOfUseAccepted) {
-          bceditNextStep = Pages.USER_PROFILE_TERMS
```

Comment:
does this has to happen only for bcied? not for bcsc

Reply:
no, bcsc as well, have the same check on that case also

Reply:
somehow it doest show up here

Reply:
may be due to that change was already there before, it only shows the changed portion in the file